coala_quickstart/generation/Project.py

```diff
@@ -29,3 +31,61 @@ def valid_path(path: StringConverter):
     if not os.path.isdir(path):
         raise ValueError("The given path doesn't exist.")
     return path
+
+
+def language_percentage(file_paths):
+    """
+    Computes the percentage composition of each language, with unknown
+    extensions tagged with the ``Unknown`` key.
+
+    :param file_paths: A list of file paths.
+    :return:           A dict with file name as key and the percentage
+                       of occurences as the value.
+    """
+    if file_paths:
+        delta = 100 / len(file_paths)
+
+    results = defaultdict(lambda: 0)
```

Comment:
why this lambda? Shouldn't it be `list`? Like the first example at https://docs.python.org/3/library/collections.html?highlight=defaultdict#defaultdict-examples